task_hive/pubspec.yaml

```
@@ -51,7 +51,7 @@ dependencies:
   intl: ^0.18.0
   mocktail: ^1.0.4
   google_sign_in: ^6.2.2
-
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)